crates/bevy_core_pipeline/src/core_3d/camera_3d.rs

```diff
@@ -15,12 +17,22 @@ use bevy_render::{
 use bevy_transform::prelude::{GlobalTransform, Transform};
 use serde::{Deserialize, Serialize};
 
-/// Configuration for the "main 3d render graph".
-/// The camera coordinate space is right-handed x-right, y-up, z-back.
+/// A 3D camera component. Enables the main 3D render graph for a [`Camera`].
+///
+/// The camera coordinate space is right-handed X-right, Y-up, Z-back.
 /// This means "forward" is -Z.
 #[derive(Component, Reflect, Clone, ExtractComponent)]
 #[extract_component_filter(With<Camera>)]
 #[reflect(Component, Default)]
+#[require(
```

Comment:
SyncToRenderWorld seems to not be added now (as far as I can tell not through any dependencies either). Is this intentional?